atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -194,4 +223,58 @@ private Column makeColumn(long ts) {
     private static ByteBuffer getRowName() {
         return ByteBuffer.wrap(PtBytes.toBytes(ROW_AND_COLUMN_NAME));
     }
+
+    private String replaceBracesWithStringFormatSpecifier(String msg) {
```

Comment:
Looks like a utils method, but I'm not sure if we still have `StringUtils` or any suitable place for it.